### ISSUE TITLE
Trip page and related functionality

### ISSUE THREAD

**Issue by samuliheljo** [HSLdevcom/digitransit-ui#107]
Implement Trip page 

![trip](https://cloud.githubusercontent.com/assets/1298522/9542285/1881ee3e-4d78-11e5-81b1-50a672807ef2.png)

General information:
- Check out route page implementation, we have quite of lot already implemented
- Before starting define togeter with Hannes the format for page url

Trip page implementation:
- [x] Ui as defined in concept
- [x] Show only current vehicle on map if realtime information is available

Route page
- [x] Click of vehicle on "road" should take user to trip page

Map functionality
- [x] Click on vehicle should open vehicle popup with "Trip page link". That should take user into trip page.

**Comment by MikaelStenstrand** [comment]
the new icons are done in separate task

**Comment by MikaelStenstrand** [comment]
the time until next stop is still not implemented
